Source/WebCore/Configurations/WebCoreSwift.xcconfig

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2009-2024 Apple Inc. All rights reserved.
```

Comment:
Why 2009?  This is a new file

Reply:
ah.. copy and paste from another config file. Fixed.